features/networking/sgw-lgw-migration.feature

```diff
@@ -98,6 +99,11 @@ Feature: SGW<->LGW migration related scenarios
     Then the step should fail
     And the output should not contain:
       | Hello OpenShift! |
+    When I run commands on the host:
+      | curl --connect-timeout 5 <%= cb.master1_ip %>:<%= cb.port %> |
```

Comment:
do we need to update this ip with braces?

Reply:
@zhaozhanqi I planned do that in separate PR if you don't mind, that will need good testing. Going to scan other node related cases as well and plans to make them compatible with IPv6.

Reply:
yes, sure.